public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx

```diff
@@ -344,7 +345,6 @@ export class CloudWatchLogsQueryField extends React.PureComponent<CloudWatchLogs
               additionalPlugins={this.plugins}
               query={query.expression ?? ''}
               onChange={this.onChangeQuery}
-              onBlur={this.props.onBlur}
```

Comment:
Should the query field also run on blur? (Assuming that there's at least one selectedLogGroup)

Reply:
@sarahzinger @iwysiu `QueryField` is configured by default to run the query on blur: https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/QueryField/QueryField.tsx#L189 Passing the prop would be needed if we wanted different behavior.

Reply:
👍  thanks for clarifying